docs/software-platforms/aws/joining-aws.md

```diff
@@ -0,0 +1,19 @@
+# Setting up AWS: Joining IAM Identity Center
+
+When we create an AWS account for you, we will also create a user for you with the username `researcher-{your github handle}`.
+
+You will receive an email from `no-reply@login.awsapps.com` with the subject "Invitation to join AWS IAM Identity Center (successor to AWS Single Sign-On)."
+
+You must accept this invitation to gain access to your AWS account and to download project data from S3.
+
+To accept the invitation, please take the following steps.
+
+1. Note the following information in the invitation email: **Your AWS access portal URL**.
+   You will need this value to [configure AWS CLI locally](../../technical-setup/environment-setup/configure-aws-cli.md) and to [log into the console to use Lightsail for Research](STUB_LINK).
```

Comment:
We should fill in this link? Maybe to the creating a VC doc? Or back to the index but specifically to the LSfR header?

Reply:
I'm filling this right now in a forthcoming PR to wrap up #343, but actually I wasn't 100% sure what link to use! I've done `./creating-vcs.md#how-to-create-a-virtual-computer` - does this make sense?

Reply:
☝️ filed as draft in #351